build.rs

```diff
@@ -1,8 +1,6 @@
 #[macro_use]
 extern crate clap;
 
-use clap::Shell;
```

Comment:
can you just comment it out rather than deleting it? Sorry for missing it before